README.md

```diff
@@ -251,18 +251,16 @@ type spaHandler struct {
 // file located at the index path on the SPA handler will be served. This
 // is suitable behavior for serving an SPA (single page application).
 func (h spaHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
-    // get the absolute path to prevent directory traversal
-	path, err := filepath.Abs(r.URL.Path)
+	var err error
```

Comment:
how would err ever get set here? we initialise it but never git it a value unless I'm misunderstanding?

Reply:
To add: err will always be nil so the error check below is unreachable code.

Reply:
OOPS! I missed it there. I think I forgot to remove the below if condition. Thank you for pointing it out.

Reply:
Addressed the above review comment.